lib/scope.js

```diff
@@ -608,9 +608,15 @@ module.exports = {
     return buffer.toString( `utf-8` );
   },
 
-  toBase64: async function ( scope, { utf8_str }) {
+  toBase64: async function ( scope, { utf8_str }, exclude_padding=false) {
+    // Sometimes, converting to base64 results in padding characters 
+    // being added to the end. Docassemble does not use these padding
+    // characters everywhere, so we allow for stripping them using 
+    // the exclude_padding argument. For more details, see
+    // https://github.com/nodejs/node/issues/6107#issuecomment-207177131
     let buffer = Buffer.from( utf8_str, 'utf-8' );
-    return buffer.toString( `base64` );
+    let base64str = buffer.toString( `base64` )
+    return exclude_padding ? base64str.replace(/[^A-Za-z0-9]/g, '') : base64str;
```

Comment:
Does docassemble use these padding characters anywhere? Maybe the default here should be `true`?

Reply:
works for me. i wasn't sure if docassemble ever kept the padding characters around.

Reply:
Sorry I didn't catch this before. It occurred to me that I may have seen characters outside of those in base64 strings in the past: https://stackoverflow.com/a/13195218/14144258. I think the safest thing to do might be to just remove any `=` in there? That would probably be `/=/g`.